rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -371,7 +573,7 @@ def _cluster_load(kubeconfig, my_path, hosted_cluster_name, mgmt_cluster_name, l
     load_env["CLEANUP_WHEN_FINISH"] = "true"
     load_env["INDEXING"] = "true"
     load_env["HYPERSHIFT"] = "true"
-    load_env["MGMT_CLUSTER_NAME"] = mgmt_cluster_name
+    load_env["MGMT_CLUSTER_NAME"] = mgmt_cluster_name + "-.*"
```

Comment:
for security reason, shall we remove thanos endpoint URL and make it read via user argument ? somehow it was overridden https://github.com/cloud-bulldozer/perfscale-managed-services/pull/107/files

Reply:
Because of how we are using the env vars, I have verified that we can just remove load_env["PROM_URL"] and load_env["THANOS_RECEIVER_URL"] and pass them during the execution of the wrapper, for example:

`PROM_URL="https://xxxxx" THANOS_RECEIVER_URL="http://xxxx" python rosa-hypershift/rosa-hosted-wrapper.py --cluster-name-seed mrnd`

Vars will be finally passed to the workload execution inside the env

Reply:
LGTM, tested - [report](https://grafana.rdu2.scalelab.redhat.com:3000/d/hcp/hypershift-report-imported?orgId=1&from=1675341458264&to=1675342210721&var-Datasource=AWS%20Pro%20-%20ripsaw-kube-burner&var-mgmt_cluster_name=hs-mc-mdqo7k4u0-.%2A&var-latencyPercentile=P99&var-uuid=5732c40e-e6e8-4533-b566-e0af9cb20b63&var-uuid=9ad4482b-2b29-40d3-99b8-0a44d3c89509&var-uuid=c32f3f99-cc99-493c-9a87-81b2c78d26b7&var-uuid=e7b73bf8-b26e-499a-9585-a750e2116517&var-sdn=OVNKubernetes&var-hosted_cluster_name=21kkg1ac5q3qg79lo6svg14o6shq93i5&var-hosted_cluster_name=21kkg2tci58b0rmpnd4v4rf55n6hmseh&var-hosted_cluster_name=21klp5asoslu3tuki00q3a0dpalfciak&var-hosted_cluster_name=21klp64vboll78ovbusip4hjnpd93a6j)